jupiterone/questions/questions.yaml

```
@@ -1,15 +1,27 @@
 ---
-sourceId: managed:template
+sourceId: managed:kandji
 integrationDefinitionId: '${integration_definition_id}'
-questions: []
-  # - id: integration-question-template-replace-me
-  #   title: What kinds of questions will this integration support?
-  #   description:
-  #     TODO Every integration should contribute questions! Please be carefule
-  #     to replace this before deploying the integration to JupiterOne.
-  #   queries:
-  #     - name: good
-  #       query: |
-  #         find * with _integrationDefinitionId = '${integration_definition_id}'
-  #   tags:
-  #     - template
\ No newline at end of file
+questions: 
+ - id: integration-question-kandji-custom-app-with-no-bundleSize-identified
+   title: Which Kandji custom applications do not have a bundleSize?
+   description:
+     Returns a list of custom Kandji applications that do not have a bundleSize limit.
```

```suggestion
     Returns a list of custom Kandji applications that do not have a bundleSize limit.
```
Is bundleSize a limit?